templates/distribution/manifests/dr/secrets/rclone.conf.tpl

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+{{- if and (index .spec.distribution.modules.dr "etcdBackup") (.spec.distribution.modules.dr.etcdBackup.s3)}}
+[minio]
+type = s3
+provider = Minio
```

Comment:
what if we want to use a real S3 bucket?

Reply:
The Minio protocol should be the same as the S3 one, so I expect nothing would change. I can test with a real S3 bucket and see if something breaks.

Reply:
Add support for `insecure` use consistency and check if AWS S3 works out-of-the-box and consistently with what we're doing with Velero.

Reply:
Added support for `insecure`. Behaviour is consistent with other places, like Velero. AWS S3 not checked yet.

Reply:
I don't see minio in rclone's docs though: https://rclone.org/docs/#configure

Maybe it is an alias for S3?

Anyway, working with S3 is a must, don't forget to check this.